doc/classes/ProjectSettings.xml

```diff
@@ -150,6 +150,7 @@
 			<param index="0" name="pack" type="String" />
 			<param index="1" name="replace_files" type="bool" default="true" />
 			<param index="2" name="offset" type="int" default="0" />
+			<param index="3" name="require_encryption" type="bool" default="false" />
 			<description>
 				Loads the contents of the .pck or .zip file specified by [param pack] into the resource filesystem ([code]res://[/code]). Returns [code]true[/code] on success.
```

Comment:
Should probably add a note that encryption is not supported for `.zip` files.

Reply:
Bump @bruvzg.